### ISSUE TITLE
parseEncryptedPassphrase and decryptPassphraseWithPassword combination should return proper error

### ISSUE THREAD

**Issue by nazarhussain** [LiskHQ/lisk-elements#688]
### Description

`lisk.default.cryptography.parseEncryptedPassphrase` when used in combination of `lisk.default.cryptography.decryptPassphraseWithPassword` is not returning valid error in case of no salt provided

### Steps to Reproduce

```
const encryptedPassphrase = 'iterations=1&cipherText=73f5827fcd8eeab475abff71476cbce3b1ecacdeac55a738bb2f0a676d8e543bb92c91e1c1e3ddb6cef07a503f034dc7718e39657218d5a955859c5524be06de5954a5875b4c7b1cd11835e3477f1d04&iv=aac6a3b77c0594552bd9c932&tag=86231fb20e7b263264ca68b3585967ca&version=1'
try {
   lisk.default.cryptography.decryptPassphraseWithPassword(
       lisk.default.cryptography.parseEncryptedPassphrase(encryptedPassphrase));
} catch(error) {
   console.log(error)
}
```
**Expected behavior:** 
I expect error `No salt provided`

**Actual behavior:** 
I expect error `Argument must be a string`. This makes it confusing for user to understand the error, which element is missing or not provided in encrypted passphrase. 

**Reproduces how often:**
Always

### Versions
`1.0.0-beta.4`

**Comment by willclarktech** [comment]
@nazarhussain How did you generate this string?

**Comment by nazarhussain** [comment]
From some tests in the core. 😄

**Comment by willclarktech** [comment]
Closed by #709
